src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
```

Comment:
Use descriptive function names.
Provide an explicit return type in a function declaration.
```suggestion
export async function getCommentsByPostId(postId: number): Promise<Comment[]> {
```